packages/forma-36-website/src/components/Layout.js

```diff
@@ -14,13 +15,10 @@ import './Layout.css';
 const styles = {
   main: css`
     display: flex;
-    flex: 2;
+    height: calc(100vh - 70px);
   `,
-  test: css`
```

Comment:
This looks like something that I did in the past! 🤪

Reply:
Tbh, I think that it is very easy to forget to remove experiments and typos in the website codebase because it needs more things like Typescript

and eslint does not seem to work properly

I will try to fix these quality of life things some day